source/scss/layout/_header.scss

```diff
@@ -6,6 +6,7 @@ body > header {
   display: block;
   margin: 0 auto;
   min-width: 320px;
+  position: sticky;
```

Comment:
I believe there isn't still much support for this feature... Please see this: https://caniuse.com/#feat=css-sticky

Reply:
Or you added this piece of code here to help with that? https://github.com/tourstream/fti-portal-styleguide/pull/116/files#diff-6e172d48821ec5d2e2138edc9f0a9bd6R51

Reply:
Exactement. 😉